Tools/autotest/arducopter.py

```diff
@@ -7850,39 +7891,173 @@ def AP_Avoidance(self):
         self.context_pop()
 
     def PAUSE_CONTINUE(self):
-        self.load_mission("copter_mission.txt", strict=False)
+        self.load_mission(filename="copter_mission.txt", strict=False)
+        self.set_parameter(name="AUTO_OPTIONS", value=3)
+        self.change_mode(mode="AUTO")
+        self.wait_ready_to_arm()
+        self.arm_vehicle()
 
-        self.set_parameter("AUTO_OPTIONS", 3)
-        self.change_mode('AUTO')
+        self.wait_current_waypoint(wpnum=3, timeout=500)
+        self.send_pause_command()
+        self.wait_groundspeed(speed_min=0, speed_max=1, minimum_duration=5)
+        self.send_resume_command()
+
+        self.wait_current_waypoint(wpnum=4, timeout=500)
+        self.send_pause_command()
+        self.wait_groundspeed(speed_min=0, speed_max=1, minimum_duration=5)
+        self.send_resume_command()
+
+        self.wait_disarmed(timeout=500)
+
+    def PAUSE_CONTINUE_GUIDED(self):
+        self.start_subtest("Started test for Pause/Continue in GUIDED mode with LOCATION!")
+        self.change_mode(mode="GUIDED")
         self.wait_ready_to_arm()
         self.arm_vehicle()
+        self.set_parameter(name="GUID_TIMEOUT", value=120)
```

Comment:
I really think this deserves a fix me comment. As it is a contrived test and needs future fixing. A timeout of 120 seconds is a very long time and is more than long enough to fall out to the sky in...... 

And we should create an issue for it so we dont forget

```suggestion
        self.set_parameter(name="GUID_TIMEOUT", value=120)
```